suggester/src/main/java/org/opengrok/suggest/Suggester.java

```diff
@@ -199,7 +199,9 @@ public void init(final Collection<NamedIndexDir> luceneIndexes) {
      * @throws InterruptedException
      */
     public void waitForInit(long timeout, TimeUnit unit) throws InterruptedException {
-        initDone.await(timeout, unit);
+        if (!initDone.await(timeout, unit)) {
+            LOGGER.log(Level.WARNING, "await timed out due to timeout");
```

Comment:
`timed out due to timeout` sounds a bit funny :D maybe something like this would be better?
`Initialization did not finish in {0} {1}, new Object[] {timeout, unit})`